app.test.js

```diff
@@ -0,0 +1,83 @@
+const request = require('supertest');
+const app = require('./app');
+const regeneratorRuntime = require('regenerator-runtime');
```

Comment:
Same as above question about babel, since we have not seen this in projects/code alongs in class, let's ask about this as well.